src/averageAges.js

```diff
@@ -37,9 +45,23 @@
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let womens = people.filter(person => person.sex === 'f');
+
+  if (withChildren === true) {
+    womens = womens.filter(woman => people.some(person =>
+      person.mother === woman.name));
```

Comment:
The condition for filtering women with children is correct, but the use of 'true' in the comparison (`withChildren === true`) is redundant. You can simplify this to `if (withChildren) { ... }`.